.gitignore

```diff
@@ -9,3 +9,4 @@ pyvenv.cfg
 
 # symlinked from submodule
 docs/volto
+/lib64
```

Comment:
This is not symlinked from submodule. Is this from a virtual environment on arch64? If so, then it should go under the `#Dependencies` heading.